h5p-bildetema-words-grid-view/package.json

```diff
@@ -44,11 +37,14 @@
     "@types/jquery": "^3.5.16",
     "@types/react": "^18.2.7",
     "@types/react-dom": "^18.2.4",
+    "eslint-config-custom": "*",
     "h5p-generate-configs": "^0.4.2",
     "sass": "^1.62.1",
     "ts-node": "^10.9.1",
-    "typescript": "^5.0.4",
-    "vite": "^4.3.9",
-    "vitest": "^0.31.1"
+    "tsconfig-custom": "*",
+    "vite-utils": "*"
+  },
+  "eslintConfig": {
+    "extends": "custom"
   }
```

Comment:
Because the eslint config would only be one line, I decided to delete the file and just place the config within package.json. Feel free to undo this at any point